bin/package-update-textdomain.js

```diff
@@ -0,0 +1,8 @@
+const wpTextdomain = require( 'wp-textdomain' );
+
+wpTextdomain( 'packages/**/*.php', {
```

Comment:
While I see this is intentionally just replacing what the old script was already doing, I wonder if this would be good to add as extra protection on WooCommerce core files too?

Reply:
@nerrad we do it already with coding sniffers, so not possible to commit if is an incorrect text domain or missing.
Also we check it while we are deploying with a grunt task https://github.com/woocommerce/woocommerce/blob/master/Gruntfile.js#L227

We can apply for all files, but currently only packages got different text domains, and are the only text domains that we can't control, different from the files on `includes/libraries` (that we also plan to remove some of those)